CONTRIBUTING.rst

```diff
@@ -105,7 +105,8 @@ To run or build the docs locally, you need to first install the required depende
 
 ``poetry install --with docs``
 
-Then you can serve the documentation with ``make docs-serve``, or build them with ``make docs``
+Then you can serve the documentation by entering the poetry with shell: ``poetry shell`` followed by ``make docs-serve``
+to serve the docs, or only build with ``make docs``.
```

Comment:
We have a whole paragraph on how to deal with the venv thing a few lines above:

https://github.com/starlite-api/starlite/pull/1382/files#diff-76e7ca2f2843ac5f659b45382768c1b64bf8704e8d3c6189c4e3623f5d465bc2R14

Reply:
ok, but seriously - this is the review you do on this PR?

Reply:
Relax please.

No, this is not my review of this PR, it’s just a comment I made because you brought this topic up on discord and I happened to notice this commit.

I’ll do a proper review tomorrow, when I have the time for it.